comit/src/network/protocols/orderbook.rs

```diff
@@ -450,7 +450,7 @@ mod tests {
             (
                 BehaviourOutEvent::TakeOrderConfirmation {
                     peer_id: alice_got_peer_id,
-                    order_id: alice_got_order_id,
+                    order_id: _,
```

Comment:
I prefer
```
commit 3aaceb8e98d1d2d0c3c1a5a3f3212cb99e21c582
Author: Tobin C. Harding <tobin@coblox.tech>
Date:   Fri Jul 17 13:57:08 2020 +1000

    Add exhaustive assertions to orderbook unit test

diff --git a/comit/src/network/protocols/orderbook.rs b/comit/src/network/protocols/orderbook.rs
index 6d0caf43f..f5e8e90c3 100644
--- a/comit/src/network/protocols/orderbook.rs
+++ b/comit/src/network/protocols/orderbook.rs
@@ -461,7 +461,10 @@ mod tests {
             ) => {
                 assert_eq!(alice_got_peer_id, bob.peer_id);
                 assert_eq!(bob_got_peer_id, alice.peer_id);
-                assert_eq!(bob_got_order_id, alice_order.id);
+
+                assert_eq!(alice_got_order_id, alice_order.id);
+                assert_eq!(bob_got_order_id, alice_got_order_id);
+
                 assert_eq!(alice_got_swap_id, bob_got_swap_id);
             }
             _ => panic!("failed to get take order confirmation"),
```
I can push this if you like

Reply:
I'll push it later on, after this merges.

Reply:
bors r+